pkg/builder/options.go

```diff
@@ -154,3 +154,21 @@ type matchEveryOwner struct{}
 func (o matchEveryOwner) ApplyToOwns(opts *OwnsInput) {
 	opts.matchEveryOwner = true
 }
+
+// InDefaultCluster configures the input to only watch objects on the default
+// cluster, even if a cluster provider is set.
+var InDefaultCluster = inDefaultCluster{}
+
+type inDefaultCluster struct{}
+
+func (f inDefaultCluster) ApplyToFor(opts *ForInput) {
+	opts.forceDefaultCluster = true
```

Comment:
Should we invest this where "false" would be default? for "single cluster"?

Reply:
In single cluster it is always the default cluster that gets the watches. In multi-cluster mode the provider clusters get the watches. This variable is to override the later. It's a noop in the former.